app/webpack/observations/identify/ducks/suggestions.js

```diff
@@ -122,7 +122,7 @@ export default function reducer(
         if ( observation.places ) {
           const place = _
             .sortBy( observation.places, p => p.bbox_area )
-            .find( p => p.admin_level !== null && p.admin_level < 3 );
+            .find( p => p.admin_level !== null && p.admin_level < 30 );
```

Comment:
Would be nice to find a better place for these constants in JS, though clearly I didn't do that when I wrote this stuff originally. Maybe a `.js.erb` that gets included in the application bundle? Something to consider in the future, maybe.

Reply:
We often define constants like SITE, CURRENT_USER, LIFE_TAXON, etc in views when rendering React components. So maybe we could extract those as well as other things we use more than once like these admin_level contstants into a JS erb file